# Copyright (C) 2025 Dan Novischi. All rights reserved.
# This software may be modified and distributed under the terms of the
# GNU Lesser General Public License v3 or any later version.
keyboard_teleoperation_node:
  ros__parameters:
    linear_velocity: 0.5 # Maximum linear velocity m/s
    angular_velocity: 1.0 # Maximum angular velocity rad/s
    key_mapping: # Commands key mapping
      forward: 'w'
      backward: 's'
      left: 'a'
      right: 'd'
      stop: 'x'
    timer_freq: 20  # Timer frequency in Hz
